essence-of-live-coding/test/Cell/Util.hs

```diff
@@ -150,3 +164,58 @@ test = testGroup "Utility unit tests"
     , output = [[0,0,0],[1,1],[2,2,0]]
     }
   ]
+
+testTraverse' :: Test
+testTraverse' = testGroup "Traversing unit tests"
+  [ testProperty "traverse' (arr f) = arr (f <$>) ([a])"
+    $ \(input :: [[Int]], Fn (f :: Int -> Int)) -> traverseArrLaw f input
```

Comment:
Instead of replicating the same test for several types, maybe you can also create types

```haskell
data TestInput = IntList [Int] | MaybeInt (Maybe Int) | VectorOfInts (Int, Int, Int, Int)
  deriving (Eq, Show, Arbitrary, CoArbitrary)
```
And then use that type for `input` and `f`.